plutus-use-cases/test/Spec/renderVesting.txt

```diff
@@ -170,25 +170,25 @@ Balances Carried Forward:
   Value:
     Ada:      Lovelace:  10000
   
-  Script: feb7e23b088a67f4f91442c3c094301b1b91d542e22c7c908ef4632cb0d66f95
+  Script: f36a17dc1fc65507e1947eb00b60730e4094890a8f829052a6e31146b386a2d0
   Value:
     Ada:      Lovelace:  60
 
 ==== Slot #13, Tx #0 ====
-TxId:       df3fafd5105726317dfb4d904556f620fa682d113f7fa081472dbe1d4377e26a
+TxId:       d2b36c0f46267e27df04480cfa846f0ac3330a3843d1972a7d69d6bdf78c828c
 Fee:        -
 Forge:      -
 Signatures  PubKey: 3d4017c3e843895a92b70aa74d1b7ebc9c982ccf...
-              Signature: 5f582063b3b8b965a50920fe350f6dcac873ec08...
+              Signature: 5f58205db3cb16e8cc29cea10fc6cd5a191217e7...
```

Comment:
Out of curiosity, how come this hash has changed, but the old hash and the new one share the same somewhat long prefix: `5f5820`? @michaelpj

Reply:
Yes, I noticed that as well, and it's not just the prefix: there are several places where hashes have quite long sequences of identical bytes in the middle as well.